astilectron.go

```diff
@@ -118,6 +118,10 @@ func validateOS(os string) error {
 	return nil
 }
 
+func ValidOSes() []string {
```

Comment:
If you are willing to take the risk of somebody modifying the array from the outside, I'll be happy to do it instead